framework/raw-aes-keyring.md

```diff
@@ -189,21 +189,21 @@ deserialize the [serialized key provider info](#key-provider-information) to obt
 
 The keyring MUST attempt to decrypt the encrypted data key if and only if the following is true:
 
-- the [ciphertext](#ciphertext) and [key provider information](#key-provider-information) are successfully deserialized.
-- the key name obtained from the encrypted data key's key provider information has a value equal to this keyring's [key name](./keyring-interface.md#key-name).
-- the key provider ID of the encrypted data key has a value equal to this keyring's [key namespace](./keyring-interface.md#key-namespace).
-- the [IV length](#iv-length) obtained from the encrypted data key's key provider information has a value equal to 12.
-- the [authentication tag length](#authentication-tag-length) obtained from the key provider information has a value equal to 128.
+- The [ciphertext](#ciphertext) and [key provider information](#key-provider-information) are successfully deserialized.
+- The key name obtained from the encrypted data key's key provider information has a value equal to this keyring's [key name](./keyring-interface.md#key-name).
+- The key provider ID of the encrypted data key has a value equal to this keyring's [key namespace](./keyring-interface.md#key-namespace).
+- The [IV length](#iv-length) obtained from the encrypted data key's key provider information has a value equal to 12.
+- The [authentication tag length](#authentication-tag-length) obtained from the key provider information has a value equal to 128.
 
 If decrypting, the keyring MUST use AES-GCM with the following specifics:
 
-- it uses the [encrypt key](#encrypted-key) obtained from deserialization as the AES-GCM input ciphertext.
-- it uses the [authentication tag](#authentication-tag) obtained from deserialization as the AES-GCM input authentication tag.
-- it uses this keyring's [wrapping key](#wrapping-key) as the AES-GCM cipher key.
-- it uses the [IV](#iv) obtained from deserialization as the AES-GCM IV.
-- it uses the serialized [encryption context](structures.md#encryption-context-1) as the AES-GCM AAD.
+- It uses the [encrypt key](#encrypted-key) obtained from deserialization as the AES-GCM input ciphertext.
+- It uses the [authentication tag](#authentication-tag) obtained from deserialization as the AES-GCM input authentication tag.
+- It uses this keyring's [wrapping key](#wrapping-key) as the AES-GCM cipher key.
+- It uses the [IV](#iv) obtained from deserialization as the AES-GCM IV.
+- It uses the serialized [encryption context](structures.md#encryption-context-1) as the AES-GCM AAD.
 
 If a decryption succeeds, this keyring MUST
 add the resulting plaintext data key to the decryption materials and return the modified materials.
 
-If no decryption succeeds, the decryption MUST NOT make any update to the decryption materials.
+If no decryption succeeds, the decryption MUST NOT make any update to the decryption materials and MUST fail.
```

Comment:
Intentionally leaving vague so we can fix this under https://github.com/awslabs/aws-encryption-sdk-specification/issues/185

`MUST fail` wording provided by @mattsb42-aws